context/test_data/sample.seg

```diff
@@ -0,0 +1,1376 @@
+'ID	chrom	loc.start	loc.end	num.mark	seg.mean
```

Comment:
Would it exercise the machinery any less well if this were 10 lines long? If a really big fixture is really necessary to demonstrate a problem, it is good to have a small factory that could generate it, so you can demonstrate what the problem is really concisely.